packages/cursorless-vscode-e2e/src/suite/fixtures/recorded/scopes/sentence/changeEverySentence.yml

```diff
@@ -0,0 +1,29 @@
+languageId: markdown
+command:
+  version: 6
+  spokenForm: change every sentence
+  action:
+    name: clearAndSetSelection
+    target:
+      type: primitive
+      modifiers:
+        - type: everyScope
+          scopeType: {type: sentence}
+  usePrePhraseSnapshot: true
+initialState:
+  documentContents: |-
+    1. **Custom subtitles** - User customizable subtitles for Talon. 
+        - [Subtitles and notifications](./core/on_phrase/subtitles_and_notifications)
```

Comment:
so were both of these sentences broken or just the first one?

Reply:
It was just one sentence